tools/module.mk

```diff
@@ -85,46 +93,35 @@ pmux_OBJS:=$(patsubst %.cpp,%.o,$(pmux_SOURCES))
 pmux: $(pmux_OBJS)
 	$(CXX11) $(pmux_LDFLAGS) $< $(pmux_LDLIBS) -o $@
 
+
 $(pmux_OBJS): %.o: %.cpp $(LIBS_BIN)
 	$(CXX11) $(CPPFLAGS) $(tools_CPPFLAGS) $(CXX11FLAGS) -c $< -o $@
 
-
 # Cdb2_dump et al. - Needs more dependencies for the cdb2_ tools
 # Cdb2_dump and others. Omit cdb2_printlog for now because it needs
 # multiple $OBJS
-cdb2_SRC:=cdb2_dump/cdb2_dump.c cdb2_stat/cdb2_stat.c cdb2_verify/cdb2_verify.c
-cdb2_OBJS:=$(patsubst %.c,tools/%.o,$(cdb2_SRC))
+cdb2_SRC:=cdb2_dump/cdb2_dump.c cdb2_stat/cdb2_stat.c cdb2_verify/cdb2_verify.c cdb2_printlog/comdb2_dbprintlog.c cdb2_printlog/cdb2_printlog.c
+BERKOBJS=berkdb/common/util_sig.o berkdb/common/util_cache.o
+cdb2_OBJS:=$(patsubst %.c,tools/%.o,$(cdb2_SRC)) $(BERKOBJS)
 
 # Dependencies for the cdb2_ tools
 cdb2_CPPFLAGS:=-I$(SRCHOME)/bdb -I$(SRCHOME)/net -I$(SRCHOME)/crc32c
 
 $(cdb2_OBJS): tools_CPPFLAGS+=$(cdb2_CPPFLAGS)
 
-
-# Since there are circular dependencies -ldb and -lschemachange must
-# go first.
-cdb2_dump: tools/cdb2_dump/cdb2_dump.o
-cdb2_stat: tools/cdb2_stat/cdb2_stat.o
-cdb2_verify: tools/cdb2_verify/cdb2_verify.o
-cdb2_dump cdb2_stat cdb2_verify:
-	$(CC) $(tools_LDFLAGS) $< $(BERKDB) $(SCHEMA) $(tools_LDLIBS) -o $@
+libcdb2util.a: $(cdb2_OBJS)
+	$(AR) $(ARFLAGS) $@ $^
 
 # Cdb2_printlog
 cdb2_printlog_SOURCES:=comdb2_dbprintlog.c cdb2_printlog.c
 cdb2_printlog_OBJS:=$(patsubst %.c,tools/cdb2_printlog/%.o,$(cdb2_printlog_SOURCES))
 
 $(cdb2_printlog_OBJS): tools_CPPFLAGS+=$(cdb2_CPPFLAGS)
 
-# Since there are circular dependencies -ldb and -lschemachange must
-# go first.
-cdb2_printlog: $(cdb2_printlog_OBJS)
-	$(CC) $(tools_LDFLAGS) $^ $(BERKDB) $(SCHEMA) $(tools_LDLIBS) -o $@
-
 # Defined in the top level makefile
-TASKS+=$(lcl_TASKS) $(tools_TASKS)
-OBJS+=tools/cdb2sql/cdb2sql.o tools/comdb2sc/comdb2sc.o			\
-tools/cdb2_sqlreplay/cdb2_sqlreplay.o $(cdb2sockpool_OBJS)		\
-$(comdb2ar_OBJS) $(pmux_OBJS) $(cdb2_OBJS) $(cdb2_printlog_OBJS)
+TASKS+=$(lcl_TASKS) $(tools_LIBS) pmux
```

Comment:
Should we add `comdb2ar` here as well? I noticed that `make clean` does not seem to clean up `comdb2ar`